ui/src/dashboards/components/TableOptions.tsx

```diff
@@ -0,0 +1,205 @@
+import React, {PureComponent} from 'react'
+import {connect} from 'react-redux'
+import {bindActionCreators} from 'redux'
+
+import GraphOptionsCustomizeFields from 'src/dashboards/components/GraphOptionsCustomizeFields'
+import GraphOptionsFixFirstColumn from 'src/dashboards/components/GraphOptionsFixFirstColumn'
+import GraphOptionsSortBy from 'src/dashboards/components/GraphOptionsSortBy'
+import GraphOptionsTimeAxis from 'src/dashboards/components/GraphOptionsTimeAxis'
+import GraphOptionsTimeFormat from 'src/dashboards/components/GraphOptionsTimeFormat'
```

Comment:
I wonder if it would be worth grouping these under a legit namespace like `GraphOptions`:

```js
import GraphOptions from 'src/dashboards/components/GraphOptions'
```

and then using like

```
<GraphOptions.TimeAxis
    ...
/>
```

Reply:
I am not sure how to best organize components into files to support this